CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 ## master / unreleased
 
+* [FEATURE] Fan out parallelizable queries to backend queriers concurrently.
+  * `-querier.sum-shards` (bool)
+  * Requires a shard-compatible schema (v10+)
+  * This causes the number of traces to increase accordingly.
+  * The query-frontend now requires a schema config to determine how/when to shard queries, either from a file or from flags (i.e. by the `config-yaml` CLI flag). This is the same schema config the queriers consume.
```

Comment:
IMHO this specific line should be a `[CHANGE]`, because the frontend won't work anymore without it.

Reply:
This should still be fine. The schema load path will automatically add a `v1` schema if overrides (config file, etc) are not specified.